Tools/scripts/build_options.py

```diff
@@ -35,8 +35,10 @@ def __init__(self,
     Feature('AHRS', 'VISUALODOM', 'HAL_VISUALODOM_ENABLED', 'Enable Visual Odometry', 0, None),
 
     Feature('Safety', 'PARACHUTE', 'HAL_PARACHUTE_ENABLED', 'Enable Parachute', 0, None),
+    Feature('Safety', 'FENCE', 'AP_FENCE_ENABLED', 'Enable Geofence', 0, None),
```

Comment:
default fence to on

Reply:
Defaulted to value of 2 to get vehicle dependent dummy methods, the same as the hard-coded default. I'm not sure if the server can handle a value of 2?